packages/codesniffer/tests/php/ci-can-run.sh

```diff
@@ -2,7 +2,6 @@
 
```

Comment:
This needs execute bits, `chmod +x packages/codesniffer/tests/php/ci-can-run.sh`

Personally I'd also put the blank line back in so there's no difference to the file text, but if you leave it removed I won't complain.